compiler/psi/src/org/jetbrains/kotlin/psi/KtStringTemplateExpression.java

```diff
@@ -86,4 +86,17 @@ public boolean hasInterpolation() {
 
         return false;
     }
+
+    // prefix"string"
+    private boolean hasPrefix() {
+        return (getChildren().length > 0 && getChildren()[0] instanceof KtNameReferenceExpression);
```

Comment:
May be we should use `getStubOrPsiChildren` instead

Reply:
There is also `getStubOrPsiChild` which does exactly what I need. Fixed